zug/transducer/dedupe.hpp

```diff
@@ -38,6 +39,8 @@ ZUG_INLINE_CONSTEXPR struct dedupe_t
                 });
         };
     }
-} dedupe{};
+};
+
+ZUG_INLINE_CONSTEXPR auto dedupe = make_composable(dedupe_t{});
```

Comment:
I do not like the use of `make_...` here, because this is just wrapping more than "making" something new. I think we can actually just use `comp(dedupe_t{})`. Same everywhere else. Actually I would remove `make_composable` itself since the unary version of `comp` does the same thing already.

Reply:
The unary version of `comp` actually just forwards the function as it currently is (without wrapping the type). I haven’t touched this since the previous implementation. I can change that, though.

Reply:
Yeah, I would do that change.